airflow/ui/src/components/DataTable/index.tsx

```diff
@@ -17,6 +17,8 @@
  * under the License.
  */
 
-import { DataTable } from "./DataTable";
 
-export { DataTable };
+
+
+
+export {DataTable} from "./DataTable";
```

Comment:
Fixed automatically by the linter, importing to export can be simplified like this.